app/src/main/java/com/android/voyageur/model/place/GooglePlacesRepository.kt

```diff
@@ -1,29 +1,62 @@
 package com.android.voyageur.model.place
 
+import com.google.android.gms.maps.model.LatLng
 import com.google.android.libraries.places.api.model.AutocompleteSessionToken
 import com.google.android.libraries.places.api.model.Place
+import com.google.android.libraries.places.api.model.RectangularBounds
+import com.google.android.libraries.places.api.model.kotlin.circularBounds
 import com.google.android.libraries.places.api.net.FetchPlaceRequest
 import com.google.android.libraries.places.api.net.FindAutocompletePredictionsRequest
 import com.google.android.libraries.places.api.net.PlacesClient
+import kotlin.math.cos
 
 class GooglePlacesRepository(private val placesClient: PlacesClient) : PlacesRepository {
+  private val placeFields =
+      listOf(
+          Place.Field.ID,
+          Place.Field.DISPLAY_NAME,
+          Place.Field.ADDRESS,
+          Place.Field.PHOTO_METADATAS,
+          Place.Field.INTERNATIONAL_PHONE_NUMBER,
+          Place.Field.WEBSITE_URI,
+          Place.Field.OPENING_HOURS,
+          Place.Field.RATING,
+          Place.Field.USER_RATINGS_TOTAL,
+          Place.Field.LAT_LNG,
+          Place.Field.PRICE_LEVEL)
+
+  private fun circularBounds(center: LatLng, radius: Double): RectangularBounds {
+    val earthRadius = 6371000.0 // Earth's radius in meters
+
+    val latDelta = radius / earthRadius * (180 / Math.PI) // Convert to degrees
+    val lngDelta =
+        radius / (earthRadius * cos(Math.toRadians(center.latitude))) *
+            (180 / Math.PI) // Adjust for latitude
+
+    val southwest = LatLng(center.latitude - latDelta, center.longitude - lngDelta)
+    val northeast = LatLng(center.latitude + latDelta, center.longitude + lngDelta)
+
+    return RectangularBounds.newInstance(southwest, northeast)
+  }
+
   override fun searchPlaces(
       query: String,
```

Comment:
Can you please add documentation for this class. We should make an effort on this matter as it came up in the individual feedback for M1

Reply:
This class was written by you though, but I will create the documentation for this.

Reply:
I have another issue regarding this class #103. The comment just included the function you completed. The documentation for the rest of the class I will do when I raise the PR for the previous issue : )

Reply:
Ok, I misunderstood your comment. Will add some context only for the new function

Reply:
Done